language-plutus-core/src/Language/PlutusCore/CBOR.hs

```diff
@@ -272,3 +285,88 @@ instance Serialise UnknownDynamicBuiltinNameError
 instance (Closed uni, uni `Everywhere` Serialise, Serialise ann) => Serialise (InternalTypeError uni ann)
 instance (Closed uni, uni `Everywhere` Serialise, Serialise ann) => Serialise (TypeError uni ann)
 instance (Closed uni, uni `Everywhere` Serialise, Serialise ann) => Serialise (Error uni ann)
+
+
```

Comment:
Maybe all of this stuff should go in its own file.

Reply:
I think it belongs here.